sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/Readme.md

```diff
@@ -23,3 +37,482 @@ var client = new TimeSeriesInsightsClient(
     tokenCredential);
 ```
 
+## Create, get, replace, and delete instances
+
+### Create instances
+You can use the following snippet to create instance Id using `TimeSeriesId`.
+
+```csharp
+TimeSeriesId instanceId = modelSettings.TimeSeriesIdProperties.Count switch
+{
+    1 => new TimeSeriesId("key1"),
+    2 => new TimeSeriesId("key1", "key2"),
+    3 => new TimeSeriesId("key1", "key2", "key3"),
+    _ => throw new Exception($"Invalid number of Time Series Insights Id properties."),
+};
+```
+
+Let's create instances using the code below. 
+```C# Snippet:TimeSeriesInsightsSampleCreateInstance
```

Comment:
It would be good to also review snippets and ensure it includes code necessary to understand for customers for ex. hardcoded strings/values for better context.